templates/landscape/index.html

```diff
@@ -1,284 +1,431 @@
+            },
+            {
+              "list_item_type": "tick",
+              "content": "通过 Landscape 还可以远程更新和升级设备，并管理用户和权限"
+            },
+          ]
+        }
+      },
+      {
+      "type": "cta-block",
+      "item": {
+          "link": {
+            "content_html": "了解 Landscape 功能 &rsaquo;",
+            "attrs": {
+              "href": "https://ubuntu.com/landscape/features",
+            }
+          }
+        }
+      },
+    ]) }}
 
-  <section class="p-strip">
-    <div class="u-fixed-width u-sv3">
-      <h2 class="u-sv3">Landscape工作机制</h2>
-      <ul class="p-list is-split">
-        <li class="p-list__item is-ticked">Landscape通过安装在每台机器上的管理Agent监控您的系统</li>
-        <li class="p-list__item is-ticked">Agent 与 Landscape 服务器通信以更新自动选择的一组基础健康指标</li>
-        <li class="p-list__item is-ticked">数据被安全地收集并存储在 Landscape 数据库中，并支持自定义指标数据收集</li>
-        <li class="p-list__item is-ticked">Landscape还使得您能远程更新和升级机器以及管理用户和权限</li>
-      </ul>
-    </div>
-  </section>
-
-  <section class="p-strip--light">
-    <div class="row">
-      <div class="col-9">
-        <h2>Landscape可以在短短三年内为拥有1000个桌面电脑的公司节省超过20万美元</h2>
-        <p>
-          <a href="https://assets.ubuntu.com/v1/0233c8a3-EMA_Canonical_Landscape_0812_WP.pdf">阅读案例分享</a>
-        </p>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2 id="savings">
+          使用 Landscape 节省成本
+        </h2>
       </div>
-    </div>
-    <div class="row">
-      <div class="col-4">
-        <div class="p-card--highlighted">
-          <h3 class="p-heading--1">1,121%</h3>
-          <p>5年以上的投资回报率</p>
-        </div>
-      </div>
-      <div class="col-4">
-        <div class="p-card--highlighted">
-          <h3 class="p-heading--1">$688000</h3>
-          <p>每年节省</p>
-        </div>
-      </div>
-      <div class="col-4">
-        <div class="p-card--highlighted">
-          <h3 class="p-heading--1">2个月</h3>
-          <p>投资回收期</p>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>一家拥有 1000 台 desktop 的公司在使用 Landscape 后，短短三年内即可节省 20 多万美元</p>
+          <div class="p-cta-block">
+            <a href="https://insights.ubuntu.com/2014/06/12/enterprise-class-ubuntu-management-with-canonical-landscape/">阅读案例研究&rsaquo;</a>
+          </div>
         </div>
       </div>
     </div>
-    <div class="u-fixed-width">
-      <p class="u-align-text--right">以上数字基于1000个桌面系统的部署。</p>
-    </div>
-  </section>
-
-  <section class="p-strip">
     <div class="row">
-      <div class="col-4 u-hide--small u-hide--medium u-vertically-center u-align--center">
-        {{
-        image (
-        url="https://assets.ubuntu.com/v1/db86a8d0-Capgemini_logo.svg",
-        alt="",
-        width="430",
-        height="95",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-        }}
+      <div class="col-9 col-medium-6 col-start-large-4">
+        <div class="p-section--shallow">
```

Comment:
I don't think it should be shallow, on the original webpage the bottom padding is 64px